be/test/olap/delete_handler_test.cpp

```diff
@@ -59,6 +61,7 @@ void set_up() {
     doris::EngineOptions options;
     options.store_paths = paths;
     doris::StorageEngine::open(options, &k_engine);
+    k_engine->set_heartbeat_flags(&heartbeat_flags);
```

Comment:
I think if change as what I suggest, this unit test does not need to be modified